### PR TITLE
Korea translate complete

### PR DESCRIPTION
**Pull request by lu1z5ec**
Added Korean.

### CODE REVIEW

File: Source/Resource/Translation/apd_ko_KR.ts [hunk 1]
```diff
@@ -5,256 +5,272 @@
...
     <message>
         <source>Unbind AirPods</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">AirPods 연결해제</translation>
     </message>
     <message>
-        <source>Launch when system starts</source>
+        <source>시스템 시작 시 실행</source>
         <translation type="unfinished"></translation>
     </message>
```
Comment (SpriteOvO) [inline Source/Resource/Translation/apd_ko_KR.ts:225]: Here is a mistake, the source text should not be modified.

File: Source/Resource/Translation/apd_ko_KR.ts [hunk 2]
```diff
@@ -5,256 +5,272 @@
...
     <message>
         <source>Language</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">언어</translation>
     </message>
     <message>
         <source>Visual</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">아이콘</translation>
     </message>
```
Comment (SpriteOvO) [inline Source/Resource/Translation/apd_ko_KR.ts:233]: You should translate the meaning as it is. Although the Visual tab currently has only one icon setting, more Visual-related settings will be added in the future.

File: Source/Resource/Translation/apd_ko_KR.ts [hunk 3]
```diff
@@ -5,256 +5,272 @@
...
     <message>
         <source>Do you want this program to launch when the system starts?
 
 If you frequently use AirPods with this computer, it is recommended that you click &quot;Yes&quot;.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">시작 시 이 프로그램을 실행 하시겠습니까?
+
+
+이 컴퓨터에서 AirPods을 자주 사용하는경우 &quot;예&quot;를 클릭 해주세요.<translation>
```
Comment (SpriteOvO) [inline Source/Resource/Translation/apd_ko_KR.ts:28]: End tag `</translation>` is missing.

File: Source/Resource/Translation/apd_ko_KR.ts [hunk 4]
```diff
@@ -5,256 +5,272 @@
...
     </message>
     <message>
         <source>Disconnected</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">연결끊김</translation>
     </message>
     <message>
         <source>Waiting for Binding</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">연결 대기중/translation>
```
Comment (SpriteOvO) [inline Source/Resource/Translation/apd_ko_KR.ts:190]: End tag `</translation>` is missing.

**Review by SpriteOvO — CHANGES_REQUESTED** [review]
Thanks! I found some issues in your commits, could you fix them?

It looks like you are not using Qt Linguist but editing the file manually. The Qt Linguist tool does help translators to translate text more correctly and efficiently. But don't worry, I will eventually help you fix those minor issues I didn't point out :)

File: Source/Resource/Translation/apd_ko_KR.ts (hunk 1 above)
Comment (lu1z5ec) [reply Source/Resource/Translation/apd_ko_KR.ts:225]: ok thank

Comment (lu1z5ec) [reply Source/Resource/Translation/apd_ko_KR.ts:225]: sorry. I have installed Qt Linguist and am fixing it again.

Comment (SpriteOvO) [reply Source/Resource/Translation/apd_ko_KR.ts:225]: @lu1z5ec There are no new changes here, you should commit your changes.

**Comment by SpriteOvO** [comment]
@lu1z5ec There are no new changes here, you should commit your changes.

**Pull request closed without merging**